packages/mdc-slider/README.md

```diff
@@ -0,0 +1,274 @@
+| `deregisterResizeHandler(handler: EventListener) => void` | Removes an event listener `handler` that was attached via `registerResizeHandler` |
+| `notifyInput() => void` | Broadcasts an "input" event notifying clients that the slider's value is currently being changed. The implementation should choose to pass along any relevant information pertaining to this event. In our case we pass along the instance of the component for which the event is triggered for. |
+| `notifyChange() => void` | Broadcasts a "change" event notifying clients that a change to the slider's value has been committed by the user. Similar guidance applies here as for `notifyInput()`. |
+| `setThumbContainerStyleProperty(propertyName: string, value: string) => void` | Sets a dash-cased style property `propertyName` to the given `value` on the thumb container element. |
+| `setTrackStyleProperty(propertyName: string, value: string) => void` | Sets a dash-cased style property `propertyName` to the given `value` on the track element |
+| `isRTL() => boolean` | True if the slider is within an RTL context, false otherwise. |
+
+#### MDCSliderFoundation API
+
+| Method Signature | Description |
+| --- | --- |
+| `layout() => void` | Same as layout() detailed within the component methods table. Does the majority of the work; the component's layout method simply proxies to this. |
+| `getValue() => number` | Returns the current value of the slider |
+| `setValue(value: number) => void` | Sets the current value of the slider |
+| `getMax() => number` | Returns the max value the slider can have |
+| `setMax(max: number) => void` | Sets the max value the slider can have |
+| `getMin() => number` | Returns the min value the slider can have |
+| `setMin(min: number) => number` | Sets the min value the slider can have |
+| `getStep() => number` | Returns the step value of the slider |
+| `setStep(step: number) => void` | Sets the step value of the slider |
+| `isDisabled() => boolean` | Returns whether or not the slider is disabled |
+| `setDisabled(disabled: boolean) => void` | Disables the slider when given true, enables it otherwise. |
+
+### Theming
+
+All thematic elements of sliders make use of the **primary theme color**, including when the
+component is used within a dark mode context.
+
+#### Setting the correct background color for disabled/off slider thumbs
+
+One tricky issue with sliders is how the thumb is supposed to look when in the disabled or
+"off" states. In these cases, certain portions of the slider's thumb and track are supposed to
+become "transparent" and reveal the background color behind it. However, this presents a problem as
+there is no elegant way to derive what the background color behind the slider should be. We could
+theoretically walk up the DOM until we found an ancestor with a set background, but that would break
+the component's encapsulation model.
+
+To solve this, you can supply a css custom property `--mdc-slider-bg-color-behind-component`. When
```

Comment:
Ahh interesting. I interpret your question as what happens if there's some non-solid background behind the slider component (e.g. a background image) where the slider element is overlayed on top of it. I'm not sure how this would be handled, but I feel like it's also a bit of an edge case?